c++/src/kj/async-win32.h

```diff
@@ -91,6 +91,9 @@ class Win32EventPort: public EventPort {
     virtual Own<IoOperation> newOperation(uint64_t offset) = 0;
     // Begin an I/O operation. For file operations, `offset` is the offset within the file at
     // which the operation will start. For stream operations, `offset` is ignored.
+
+    virtual void stopObserving() = 0;
```

Comment:
Hmm this doesn't feel right. Good RAII style would have the destructor do this automatically.

I think it'd be OK to have the destructor always remove the handle (even if it's about to be closed anyway). Then we end up with similar behavior to `UnixEventPort::FdObserver`.

We can then remove the `WARNING` in the doc comment for `observeIo()`, below.